src/Outputter.ts

```diff
@@ -57,9 +61,39 @@ export class Outputter extends EventEmitter {
 	 * @param text The stdout data in question
 	 */
 	write(text: string) {
+		this.processSigilsAndWriteText(text);
+		
+	}
+	
+	/**
+	 * Add a segment of stdout data to the outputter,
+	 * processing `toggleHtmlSigil`s along the way.
+	 * `toggleHtmlSigil`s may be interleaved with text and HTML
+	 * in any way; this method will correctly interpret them.
+	 * @param text The stdout data in question
+	 */
+	private processSigilsAndWriteText(text: string) {
+		//Loop around, removing HTML toggling sigils
+		while (true) {
+			let index = text.indexOf(TOGGLE_HTML_SIGIL);
+			if (index == -1) break;
```

Comment:
Could use strict equality everywhere instead of regular equality

Reply:
Fixed; thank you!